.vscode/toast.code-snippets

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
smart ;)

Reply:
But idk if we have to commit vscode snippets ?

Reply:
I'd rather do it since they're pretty handy and don't bother the devs that don't use vscode